tensor2tensor/utils/bleu_hook.py

```diff
@@ -158,6 +159,7 @@ def property_chars(self, prefix):
     return "".join(six.unichr(x) for x in range(sys.maxunicode)
                    if unicodedata.category(six.unichr(x)).startswith(prefix))
 
+uregex = UnicodeRegex()
```

Comment:
why is this top level now? would prefer to leave it where it is

Reply:
Creating the `UnicodeRegex` instance is quite slow because I need to iterate twice over all Unicode characters.
It is really a bad idea to create a new instance for each line of the test set to be tokenized.
It caused the whole `t2t-bleu.py` to be about 10000 times slower (depending on the test set size).
As I note in my [commit log](https://github.com/tensorflow/tensor2tensor/pull/524/commits/747d4294ebe6e65e9e93649126ec3716fe300f3a), I would slightly prefer my original implementation with a class with static methods only (the class was there rather as a namespace).

Reply:
ah ok. thanks. fine as is then.